README.md

```diff
@@ -237,31 +237,31 @@ The following are valid user configuration options.
 
 Section           | Option               | Value
 ----------------- | -------------------- | -----
-Accounts          | deprovision_remove   | `true` makes deprovisioning a user destructive.
+Accounts          | deprovision\_remove   | `true` makes deprovisioning a user destructive.
 Accounts          | groups               | Comma separated list of groups for newly provisioned users.
-Accounts          | useradd_cmd          | Command string to create a new user.
-Accounts          | userdel_cmd          | Command string to delete a user.
-Accounts          | usermod_cmd          | Command string to modify a user's groups.
-Accounts          | groupadd_cmd         | Command string to create a new group.
-Daemons           | accounts_daemon      | `false` disables the accounts daemon.
-Daemons           | clock_skew_daemon    | `false` disables the clock skew daemon.
-Daemons           | ip_forwarding_daemon | `false` disables the IP forwarding daemon.
-InstanceSetup     | optimize_local_ssd   | `false` prevents optimizing for local SSD.
-InstanceSetup     | network_enabled      | `false` skips instance setup functions that require metadata.
-InstanceSetup     | set_boto_config      | `false` skips setting up a boto config.
-InstanceSetup     | set_host_keys        | `false` skips generating host keys on first boot.
-InstanceSetup     | set_multiqueue       | `false` skips multiqueue driver support.
-IpForwarding      | ethernet_proto_id    | Protocol ID string for daemon added routes.
-IpForwarding      | ip_aliases           | `false` disables setting up alias IP routes.
-IpForwarding      | target_instance_ips  | `false` disables internal IP address load balancing.
-MetadataScripts   | run_dir              | String base directory where metadata scripts are executed.
+Accounts          | useradd\_cmd          | Command string to create a new user.
+Accounts          | userdel\_cmd          | Command string to delete a user.
+Accounts          | usermod\_cmd          | Command string to modify a user's groups.
+Accounts          | groupadd\_cmd         | Command string to create a new group.
+Daemons           | accounts\_daemon      | `false` disables the accounts daemon.
+Daemons           | clock\_skew\_daemon    | `false` disables the clock skew daemon.
+Daemons           | ip\_forwarding_daemon | `false` disables the IP forwarding daemon.
+InstanceSetup     | optimize\_local_ssd   | `false` prevents optimizing for local SSD.
```

Comment:
Fwiw, this is how some markdown clients render the table (without the patch):
![image](https://user-images.githubusercontent.com/2119212/29538923-d9c4b790-8695-11e7-83d5-30b9257ca20b.png)